plover/messages/nl/LC_MESSAGES/plover.po

```diff
@@ -0,0 +1,1021 @@
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:110
+msgid "Clear"
+msgstr "Leegmaken"
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:112
+msgid "&Delete"
+msgstr "&Verwijderen"
+
+#. Widget: “DictionaryEditor”, tooltip.
+#: plover/gui_qt/dictionary_editor_ui.py:114
+msgid "Delete selected entries."
+msgstr "Verwijder de geselecteerde regels."
+
+#. Widget: “DictionaryEditor”, tooltip.
+#: plover/gui_qt/dictionary_editor_ui.py:120
+msgid "Undo last add/delete/edit operation."
+msgstr "Maak de laatste toevoeg-/verwijder-/bewerk-operatie ongedaan."
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:124
+msgid "&New translation"
+msgstr "&Nieuwe vertaling"
+
+#. Widget: “LookupDialog”, tooltip.
+#: plover/gui_qt/lookup_dialog.py:13
+msgid "Search the dictionary for translations."
+msgstr "Zoek in het woordenboek naar vertalingen."
+
+#: plover/gui_qt/lookup_dialog.py:15
+msgid "Lookup"
+msgstr "Opzoeken"
+
+#. Widget: “LookupDialog”, windowtitle.
+#: plover/gui_qt/lookup_dialog_ui.py:56
+msgid "Plover: Dictionary Lookup"
+msgstr "Plover: Zoek in woordenboek"
+
+#: plover/gui_qt/machine_options.py:104
+msgid ""
+"Arpeggiate allows using non-NKRO keyboards.\n"
+"\n"
+"Each key can be pressed separately and the\n"
+"space bar is pressed to send the stroke."
+msgstr ""
+"Door te arpeggiëren is het mogelijk om een\n"
+"niet-NKRO-toetsenbord te gebruiken.\n"
+"\n"
+"Iedere toets kan één voor één ingedrukt worden,\n"
+"waarna de spatiebalk de aanslag verzendt."
+
+#. Widget: “MainWindow”, tooltip.
+#: plover/gui_qt/main_window_ui.py:172 plover/gui_qt/main_window_ui.py:212
+msgid "Disconnect and reconnect the machine."
+msgstr "Verbreek en herstel vervolgens de verbinding met de machine."
+
+#. Widget: “MainWindow”, text.
+#: plover/gui_qt/main_window_ui.py:178
+msgid "Disabled"
+msgstr "Uitgeschakeld"
+
+#. Widget: “MainWindow”, title.
+#: plover/gui_qt/main_window_ui.py:180
+msgid "&File"
+msgstr "&Bestand"
+
+#. Widget: “MainWindow”, title.
+#: plover/gui_qt/main_window_ui.py:182
+msgid "&Tools"
+msgstr "&Gereedschappen"
+
+#. Widget: “MainWindow”, title.
+#: plover/gui_qt/main_window_ui.py:184
+msgid "&Help"
+msgstr "&Help"
+
+#. Widget: “MainWindow”, title.
+#: plover/gui_qt/main_window_ui.py:186
+msgid "&Edit"
+msgstr "&Bewerken"
+
+#. Widget: “MainWindow”, windowtitle.
+#: plover/gui_qt/main_window_ui.py:188
+msgid "Plover: Toolbar"
+msgstr "Plover: Werkbalk"
+
+#. Widget: “MainWindow”, text.
+#: plover/gui_qt/main_window_ui.py:190
+msgid "&Quit Plover"
+msgstr "Plover &afsluiten"
+
+#. Widget: “MainWindow”, tooltip.
+#: plover/gui_qt/main_window_ui.py:192
+msgid "Quit the application."
+msgstr "Sluit deze applicatie af."
+
+#. Widget: “MainWindow”, shortcut.
+#: plover/gui_qt/main_window_ui.py:194
+msgid "Ctrl+Q"
+msgstr "Ctrl+Q"
+
+#. Widget: “MainWindow”, text.
+#: plover/gui_qt/main_window_ui.py:196
+msgid "&Configure"
+msgstr "&Instellingen"
```

Comment:
"&Instellen"?
het is niet "&Configurations"
(ik ken de context niet, "&settings" kan bedoelt zijn)

Reply:
This one makes more sense in context:
![image](https://user-images.githubusercontent.com/7533280/115120198-77eacd00-9fac-11eb-83e9-09ebd5879cde.png)